src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -705,6 +704,81 @@ public void testReadBasedAnnotations() throws IOException {
         Assert.assertFalse(vc.get().getFilters().contains(GATKVCFConstants.STRAND_ARTIFACT_FILTER_NAME));
     }
 
+
+    public File createSamWithNsandStrandBias(final int numAlts, final int numNs, final int numRefs) throws IOException {
+        final byte altQuality = 50;
+        final byte refQuality = 30;
+        final File samFile = File.createTempFile("duplex", ".bam");
+        final SAMFileHeader samHeader = M2TestingUtils.createSamHeader();
+        final SAMFileGATKReadWriter writer = M2TestingUtils.getBareBonesSamWriter(samFile, samHeader);
+
+        // create 2 Ts with strand bias
```

Comment:
I don't see the 2Ts here.... am I missing something or is this a spurious comment?

Reply:
that was leftover from before...